Sources/Plasma/PubUtilLib/plPipeline/hsG3DDeviceSelector.cpp

```diff
@@ -349,23 +351,12 @@ void hsG3DDeviceSelector::Enumerate(hsWindowHndl winRef)
     IClear();
 
 #ifdef PLASMA_PIPELINE_DX
-    /// 9.6.2000 - Create the class to use as our temporary window class
-    WNDCLASS    tempClass;
-
-    strcpy( fTempWinClass, "DSTestClass" );
-    memset( &tempClass, 0, sizeof( tempClass ) );
-    tempClass.lpfnWndProc = DefWindowProc;
-    tempClass.hInstance = GetModuleHandle(nullptr);
-    tempClass.hbrBackground = (HBRUSH)GetStockObject( BLACK_BRUSH );
-    tempClass.lpszClassName = fTempWinClass;
-    uint16_t ret = RegisterClass(&tempClass);
-    hsAssert(ret, "Cannot create temporary window class to test for device modes" );
-
     ITryDirect3DTnL(winRef);
-
-    /// Get rid of the class
-    UnregisterClass(fTempWinClass, GetModuleHandle(nullptr));
 #endif
+
+    for (auto enumerator : sEnumerators) {
```

Comment:
```suggestion
    for (const auto& enumerator : sEnumerators) {
```

Avoid copying a function object